src/ipc/ipc3/handler.c

```diff
@@ -515,10 +515,14 @@ static int ipc_glb_stream_message(uint32_t header)
 		return ipc_stream_pcm_params(header);
 	case SOF_IPC_STREAM_PCM_FREE:
 		return ipc_stream_pcm_free(header);
-	case SOF_IPC_STREAM_TRIG_START:
-	case SOF_IPC_STREAM_TRIG_STOP:
 	case SOF_IPC_STREAM_TRIG_PAUSE:
 	case SOF_IPC_STREAM_TRIG_RELEASE:
+#if CONFIG_BAYTRAIL || CONFIG_CHERRYTRAIL || CONFIG_HASWELL || CONFIG_BROADWELL
```

Comment:
The aim here is to remove this code for all platforms as PUASE is upper layer feature done by pipewire, pulseaudio and CRAS. i.e. the PR should be removing code for PAUSE/RELEASE.